style/styles.css

```diff
@@ -44,7 +44,7 @@ body {
 	font-size: 18px;
 	line-height: 1.4rem;
 }
-h1, h2, h3, li {
+h1, h2, h3 {
```

Comment:
i prefer the san-serif typeface prior to this change to serif - applies only to contact info for the councillors